Apps/Src/Tasks.c

```diff
@@ -227,23 +287,97 @@ void Task_BatteryBalance(void *p_arg) {
     }
 }
 
+void Check_ResourceForCLI(int resourceNum){
```

Comment:
What is the difference between this and Pend_ResourceForCLI?

Reply:
It's the same function, I just renamed it from "Check_" to "Pend_". I will remove this.